test/commands/generate_test.rb

```diff
@@ -158,4 +158,74 @@ def default_options
       end
     end
   end
+
+  describe 'entity' do
+    let(:target)      {:entity}
```

Comment:
the style guideline is to have space in between `{}`, that is `let(:target)      { :entity }`, please change it for other `let` too.